src/scancode/utils.py

```diff
@@ -59,13 +59,21 @@ class BaseCommand(click.Command):
     # "Try 'scancode --help' for help on options and arguments."
     short_usage_help = ''
 
+    options = []
```

Comment:
The idea is since the `BaseCommand` class is extended by both `ScanCommand` and `ExtractCommand`, the `get_params` method utilizes the `self.options` attribute for both of them. It's the same reason why `short_usage_help` isn't a class level attribute I believe.